README.md

```diff
@@ -81,6 +81,7 @@
 
 - [Vlang the Game](https://github.com/alula/vlang-the-game) - A little platformer written in V with [vSDL2](https://github.com/nsauzede/vsdl2), made purely for memes and fun.
 - [Boundstone](https://github.com/organization/boundstone) - High Performance / Fast Compilation / Lightweight Minecraft: Bedrock Edition Server.
+- [v-pong](https://github.com/thebigsmileXD/v-pong) - A classic paddle game brought back to life through the power of V.
```

Comment:
Please solve

```
✖  1:1  Invalid URL: http:#applications  remark-lint:double-link
```

Reply:
Looks like the linter got a problem here, but doesn't seem to be caused by me since i only added 1 line?
#applications is an anchor and i have no idea why that would just pop up out of nowhere